rest/taskana-rest-spring/src/main/java/pro/taskana/rest/TaskanaEngineController.java

```diff
@@ -66,7 +70,6 @@
      */
     @GetMapping(path = "/v1/version", produces = {MediaType.APPLICATION_JSON_VALUE})
     public ResponseEntity<VersionResource> currentVersion() {
-        String version = TaskanaEngineController.class.getPackage().getImplementationVersion();
         if (version == null) {
             version = "1.0-DEFAULT";
```

Comment:
in what cases don't we have a version set? Maybe we can set a more meaningful message for such case? (like "local build" for example?)

Reply:
It's a good idea